tests/integration/test_data_api.py

```diff
@@ -223,6 +224,43 @@ async def test_search_filter(item_descriptions,
     assert items_list == item_descriptions
 
 
+@respx.mock
+@pytest.mark.anyio
+async def test_search_filter_positional_args(item_descriptions,
```

Comment:
nit: add a short docstring for this test.

Reply:
Also, compared to the below tests it looks like this tests 'quick-search' and pagination rather than vanilla search?  If so what do you think about renaming this test to reflect quick-search + pagination?

Reply:
Pagination removed from test and docstring added